src/Gizmos/babylon.axisDragGizmo.ts

```diff
@@ -5,6 +5,15 @@ module BABYLON {
     export class AxisDragGizmo extends Gizmo {
         private _dragBehavior:PointerDragBehavior;
         private _pointerObserver:Nullable<Observer<PointerInfo>> = null;
+        /**
+         * Drag distance in babylon units that the gizmo will snap to when dragged (Default: 0)
+         */
+        public snapDistance = 0;
+        /**
+         * Event that fires each time the gizmo snaps to a new location.
+         * * snapDistance is the the change in distance
+         */
+        public onSnapObservable = new Observable<{snapDistance:number}>();

```

Comment:
i am not seeing the clear in dispose but on a mobile :-)

Reply:
added clear